pyproject.toml

```diff
@@ -99,7 +99,7 @@ ci = ["pytest-cov>=4", "pytest-split>=0.8", "pytest>=8"]
 docs = ["invoke", "sphinx", "sphinx_markdown_builder", "sphinx_rtd_theme"]
 optional = [
     "ase>=3.23.0",
-    "beautifulsoup4",
+    "beautifulsoup4>=4.6.0",
```

Comment:
Now all dependencies in ci,optional have a lower bound, [but still get](https://github.com/materialsproject/pymatgen/pull/4073/checks):
```
warning: The direct dependency `pymatgen` is unpinned. Consider setting a lower bound when using `--resolution-strategy lowest` to avoid using outdated versions.
warning: The direct dependency `pymatgen[ci]` is unpinned. Consider setting a lower bound when using `--resolution-strategy lowest` to avoid using outdated versions.
warning: The direct dependency `pymatgen[optional]` is unpinned. Consider setting a lower bound when using `--resolution-strategy lowest` to avoid using outdated versions.
``

Reply:
Resolved